### DISCUSSION TITLE
Is there work being done on ignore attribute ? If yes, what still needs to be done ?

### DISCUSSION THREAD

**Discussion by DontBreakAlex** [tokio-rs/prost#732] [General]
While migrating code from serde_json I encountered this issue: https://github.com/tokio-rs/prost/issues/174.
In some places, I use `#[serde(skip)]`, which is not currenctly possible with prost. While browsing the code, I found several references to ignore attributes, like this https://github.com/tokio-rs/prost/blob/6658ab4de78e4ddeced7f79e1465f960de0a616f/prost-derive/src/field/mod.rs#L33-L37.
Is there work being done on this ?
If yes, what remains to be done ?
If no, how much work would be needed to implement this ?

**Comment by LucioFranco** [comment]
Hi, I don't actually know how much work but I doubt its an incredible amount of work since I imagine you just need to feed some config through. I think we could support this but beyond that I don't have much info. If you're interested I would accept a PR that adds this and can mentor a bit via discord in the discord.gg/tokio #prost channel.

**Reply by DontBreakAlex** [reply]
Ok then ! I'll take a look next weekend and drop a message on discord If I need information

**Comment by DontBreakAlex** [comment]
I dug a bit deeper and found this [commit](https://github.com/tokio-rs/prost/pull/493/commits/ba934cd126854c46d982af44f92b5133e66068ad) in @r3v2d0g's [PR](https://github.com/tokio-rs/prost/pull/493). Do you think I can cherry-pick it to get it merged ?

**Reply by LucioFranco** [reply]
Yeah, will need some thorough tests though. But the commit looks small and sizable that we can include it easily.
